dist/openshift/cincinnati.yaml

```diff
@@ -131,6 +136,11 @@ objects:
                     configMapKeyRef:
                       key: pe.mandatory_client_parameters
                       name: cincinnati
+                - name: "RUST_BACKTRACE"
+                  valueFrom:
+                    configMapKeyRef:
+                      key: pe.rust_backtrace
+                      name: cincinnati
```

Comment:
This is temporary right?

Reply:
This is required for PR e2e. This won't be set on stage or prod